packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -182,10 +193,32 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
           ariaSelected: selectedIndices[0] === undefined ? undefined : true
         };
 
+    this._classNames = getClassNames(propStyles, {
+      theme,
+      className,
+      hasError: Boolean(errorMessage && errorMessage.length > 0),
+      isOpen,
+      required,
+      disabled,
+      isRenderingPlaceholder: !selectedOptions.length,
+      panelClassName: !!panelProps ? panelProps.className : undefined,
+      calloutClassName: !!calloutProps ? calloutProps.className : undefined
+    });
+
+    const labelStyles = this._classNames.subComponentStyles
+      ? (this._classNames.subComponentStyles.label as IStyleFunctionOrObject<ILabelStyleProps, ILabelStyles>)
+      : undefined;
+
     return (
       <div className={css('ms-Dropdown-container')}>
         {label && (
-          <Label className={css('ms-Dropdown-label')} id={id + '-label'} htmlFor={id} required={required}>
+          <Label
+            className={css('ms-Dropdown-label')}
```

Comment:
Can this be defined in `GlobalClassNames` and added as part of a default labelStyles function (as part of `getStyles`) that can be disabled like the other Dropdown global class names?